instructor/patch.py

```diff
@@ -222,7 +222,8 @@ async def new_create_async(
         response_model, new_kwargs = handle_response_model(
             response_model=response_model, mode=mode, **kwargs
         )
-        new_kwargs["messages"] = handle_templating(new_kwargs["messages"], context)
+        if "messages" in new_kwargs:
```

Comment:
gemini has a kwargs of "content" so this throws an error for gemini with the new jinja support